src/Fmt_ast.ml

```diff
@@ -2908,7 +2986,8 @@ and fmt_constructor_declaration c ctx ~first ~last:_ cstr_decl =
       ( hovbox 2
           ( hvbox 2
               ( Cmts.fmt c loc
-                  (wrap_if (is_symbol_id txt) "( " " )" (str txt))
+                  (wrap_if (is_symbol_id txt) "( " " )"
+                     (str txt $ fmt_padding))
```

Comment:
The padding is before the closing parenthesis of symbol constructors:

```ocaml
type t =
  | ( ::               ) of a * b
  | Long_constructor of loooooooooooooooooooooooong_break
```

Reply:
Fixed.